packages/apps/src/contexts/activity.ts

```diff
@@ -178,13 +178,19 @@ export class ActivityContext<T extends Activity = Activity> implements IActivity
   }
 
   async send(activity: ActivityLike) {
-    return await this._plugin.send(toActivityParams(activity), this.ref);
+    const ref = {
+      ...this.ref,
+      conversation: {
```

Comment:
we may want to move this logic to the http plugin, there is already some manipulation happening there and it would be nice to have it all in one place. This logic is also highly teams specific (how the id is being structured), this logic may break compatibility with webchat/sharepoint and leadership has hinted that we may want to support those in the long term.